src/libraries/System.Net.Quic/src/Resources/Strings.resx

```diff
@@ -57,8 +117,10 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-
+  <data name="net_quic_notsupported" xml:space="preserve">
+    <value>QUIC is not supported on this platform. See http://aka.ms/dotnetquic</value>
```

Comment:
The link redirects to https://github.com/dotnet/runtime
Is this intentional or is the redirect not setup right now?

Reply:
Intentional; Around release we'll have a docs page or blog post outlining how to play with HTTP/3 -- it'll get updated at that time.